styles2.css

```diff
@@ -443,3 +443,122 @@ Button interactions
   background: #36b37f;
   border-radius: 4px;
 }
+
+/*
+About me and contact form
+*/
+.aboutme_form {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.formp {
+  width: 327px;
+  height: 363px;
+  margin-top: 60px;
+  margin-bottom: 24px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.about_form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.formname {
+  margin-top: 58px;
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  padding: 1px 15px;
+}
+
+.formemail {
+  margin-top: 24px;
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  padding: 1px 15px;
+}
+
+.textholder {
+  margin-top: 24px;
+  padding: 12px;
+  width: 327px;
+  height: 114px;
+  font-size: 15px;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #60c095;
+}
+
+.submit_but {
```

Comment:
- [ ] Button interactions such as hover, active, etc, are missing, kindly provide one.

Reply:
Done. Sorry, I thought it was done. I have added a hover effect like in the rest of the buttons.